main.tf

```diff
@@ -30,19 +56,11 @@ module pull_secret {
   docker_password = var.entitlement_key
   docker_server   = "cp.icr.io"
   secret_name     = "ibm-entitlement-key-s"
-}
+}*/
 
-resource null_resource create_yaml {
-  provisioner "local-exec" {
-    command = "${path.module}/scripts/create-yaml.sh '${local.name}' '${local.yaml_dir}'"
 
-    environment = {
-      VALUES_CONTENT = yamlencode(local.values_content)
-    }
-  }
-}
 
-module "service_account" {
+/*module "service_account" {
```

Comment:
If this is not needed, please remove the commented out code.

Reply:
removed commented code